calico-enterprise/observability/elastic/flow/aggregation.mdx

```diff
@@ -18,55 +18,53 @@ aggregation is suitable for your implementation.
+type minimizes the flow logs generated for applications that make many connections to the same destination. The two flows originating from `client-a` without aggregation are combined into one.
 
-In Kubernetes, pod controllers (Deployments, DaemonSets, ReplicaSets etc.) can automatically create names for pods. For example, the pods `nginx-1` 
-and `nginx-2` are created by the ReplicaSet `nginx`. The controller's name is considered a pod-prefix and is used to aggregate flow log entries 
-(indicated with an asterisk * at the end of the name). Flow logs originating from pods with the same prefix will be aggregated as long as the traffic 
-is on the same protocol, and destined towards the same IP, and destination port. The three flow logs without aggregation originating from `client-a` 
-and `client-b` are combined into a single flow log. This aggregation level is called `AnyConnectionFromSameSourcePodPrefix`.
+In Kubernetes, pod controllers (Deployments, DaemonSets, ReplicaSets etc.) automatically name the pods they create with a common prefix.
+For example, the pods `nginx-1` and `nginx-2` are created by the ReplicaSet `nginx`. That prefix is used to aggregate flow log entries and is indicated with an asterisk * at the end of the name. The three flows originating from `client-a` and `client-b` are combined into a single flow log. This aggregation level is called `AggregateIPsAndSourcePorts`.
 
-Finally, with `AnyConnectionBetweenSamePodPrefixes` we combine source and destination pods that are part of the same pod controller. With level 3, the flow logs
-are aggregated by the destination port and protocol, as long as they originate from pods with the same pod-prefix and destined for pods of the same 
-pod-prefix. Previously distinct logs are aggregated into a single flow log (see the last row).
+Finally, `AggregateIPsAndPorts` combines flows between matching pod prefixes that target distinct destination ports, as seen in the last example row below.
+Destination port information is often valuable, so this level should only be used after log analysis indicates it will be necessary and beneficial. 
+
+Note that currently aggregation is performed on a per node basis, but this behavior may change in future.  Certain other fields are always kept separate
+and not aggregated together such as `action` (i.e. denied traffic and allowed traffic will always be logged separately).
 
 |                          |           | **Src Traffic**  | |          | **Dst Traffic**  | |          | **Packet counts**      | |
 |--------------------------|-----------|----------|---------|----------|----------|---------|----------|------------|-------------|
 | **Aggr lvl**             | **Flows** | **Name** | **IP**  | **Port** | **Name** | **IP**  | **Port** | **Pkt in** | **Pkt out** |
 | 0 (no aggr)              | 4         | client-a | 1.1.1.1 | 45556    | nginx-1  | 2.2.2.2 | 80       | 1          | 2           |
```

Comment:
I think we should have all the flows here in level 0 and then each of the higher levels show how the flows are combined based on the packet counts. With that said I don't see the Source client-c 1.1.3.3 flow in level 0.

Reply:
The original table didn't have this info either. So you didn't leave it out, it wasn't there in the first place.

Reply:
Ah oops, I think there was a typo on the IP address in one of the lines